File: projectBlueWater/src/main/java/com/lasthopesoftware/bluewater/client/playback/nowplaying/broadcasters/notification/building/BuildPlaybackStartingNotification.java
package com.lasthopesoftware.bluewater.client.playback.nowplaying.broadcasters.notification.building;

import androidx.core.app.NotificationCompat;

import com.namehillsoftware.handoff.promises.Promise;

public interface BuildPlaybackStartingNotification {
	Promise<NotificationCompat.Builder> promisePreparedPlaybackStartingNotification();
}

File: projectBlueWater/src/main/java/com/lasthopesoftware/bluewater/client/playback/nowplaying/broadcasters/notification/building/BuildPlaybackStartingNotification.kt
package com.lasthopesoftware.bluewater.client.playback.nowplaying.broadcasters.notification.building

import androidx.core.app.NotificationCompat
import com.lasthopesoftware.bluewater.client.browsing.library.repository.LibraryId
import com.namehillsoftware.handoff.promises.Promise

interface BuildPlaybackStartingNotification {
    fun promisePreparedPlaybackStartingNotification(libraryId: LibraryId): Promise<NotificationCompat.Builder>
}
